packages/pigment-css-react/src/Grid.jsx

```diff
@@ -150,6 +156,7 @@ const Grid = React.forwardRef(function Grid(
     unstable_parent_column_spacing,
     // eslint-disable-next-line react/prop-types
     unstable_parent_row_spacing,
+    wrap = 'wrap',
```

Comment:
```suggestion
    wrap,
```

There should be no default value for this, otherwise the non-container Grid will have `flexWrap` which is a different behavior compared to the previous version.

Grid container already contain the `flexWrap` by default (see the first variant).

Reply:
I was wrong.